whylog/constraints/__init__.py

```diff
@@ -1,42 +1,48 @@
-from abc import ABCMeta, abstractmethod
+from abc import ABCMeta
 
 import six
 
 
 @six.add_metaclass(ABCMeta)
 class AbstractConstraint(object):
-    @abstractmethod
-    def save(self, rulebase_rule):
-        pass
+    pass
 
 
 class TimeConstraint(AbstractConstraint):
-    def __init__(self, line_earlier, line_later, min_delta=None, max_delta=None):
+    def __init__(self, params):
```

Comment:
Why this constructor has param params? Everything to verify constraints gets in verify() invocation